app/controllers/characters_controller.rb

```diff
@@ -1,17 +1,21 @@
 class CharactersController < ApplicationController
-  before_action :authenticate_user!
+  protect_from_forgery prepend: true
```

Comment:
ActionController::InvalidAuthenticityToken (ActionController::InvalidAuthenticityToken)
we looked it up online